common/proto/eqservice.proto

```diff
@@ -5,23 +5,25 @@ service Inclusion {
     rpc GetKeccakInclusion(GetKeccakInclusionRequest) returns (GetKeccakInclusionResponse);
 }
 message GetKeccakInclusionRequest {
-    bytes commitment = 1;  // 32 byte blob commitment
-    bytes namespace = 2;   // 32 byte namespace
-    uint64 height = 3;     // block height
+    uint64 height = 1;     // Data Avaliblity block height
+    bytes namespace = 2;   // 32 byte DA namespace
+    bytes commitment = 3;  // 32 byte DA blob commitment
 }
 
 message GetKeccakInclusionResponse {
     enum Status {
-        WAITING = 0;
-        IN_PROGRESS = 1;
-        COMPLETE = 2;
-        FAILED = 3;
+        DA_PENDING = 0;        // Data Avaliblity (DA) inclusion proof being collected
+        DA_AVALIBLE = 1;       // DA inclusion proof collected
+        ZKP_PENDING = 2;       // Zero Knowledge Proof (ZKP) of DA inclusion requested, generating
+        ZKP_FINISHED = 3;      // ZKP of DA inclusion proof finished
+        RETRYABLE_FAILURE = 4; // If this is returned, the service then attempts to retry
+        PERMANENT_FAILURE = 5; // No way to complete request
```

Comment:
breaking RPC change, to match internal & reported status. Would be great to use this (generated) enum being matched in some more compile-time enforced way :thinking: